README.md

```diff
@@ -452,40 +464,84 @@ data:
   tokenurl: "<auth_url>"
   tokenurlsuffix: "/oauth/token"
 ```
+with TLS configurations:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sap-btp-service-operator-tls
```

Comment:
@I065450 @kerenlahav shall the secret name contain '-tls' suffix??

Reply:
the name must be sap-btp-service-operator-tls